tsconfig.json

```diff
@@ -1,19 +1,30 @@
 {
   "compilerOptions": {
-    "target": "es6",
-    "lib": ["es2015", "es2017", "dom"],
+    "lib": ["es2015", "es2017", "es2020", "dom"],
+    "target": "es2017",
     "module": "commonjs",
+    "moduleResolution": "node",
     "sourceMap": true,
     "outDir": "./bin/.ts",
     "rootDir": "./",
-    "noEmit": true,
+
     "strict": true,
     "noImplicitAny": false,
+    "strictNullChecks": false,
+    "strictPropertyInitialization": false,
```

Comment:
https://www.typescriptlang.org/tsconfig#strictPropertyInitialization
Why set `strictPropertyInitialization` to false ?